spec/lib/miq_automation_engine/service_methods/miq_ae_service_classification_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe MiqAeMethodService::MiqAeServiceClassification do
+  before do
+    @cat1 = FactoryGirl.create(:classification_department_with_tags)
+    @cat2 = FactoryGirl.create(:classification_cost_center_with_tags)
+    @cat_array = Classification.categories.collect(&:name)
+    @tags_array = @cat2.entries.collect(&:name)
```

Comment:
There is reason to have these to `_array` variables here, just do the collect calls in the tests.